package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "parasoft-findings-bitbucket",
+  "version": "1.0.0",
+  "description": "A script for uploading results of Parasoft static analysis report to BitBucket.",
+  "main": "lib/main.js",
```

Comment:
Since this is a CLI-only tool and not intended to be imported as a library, we don't need the `main` field in `package.json`. It can be safely removed to avoid confusion about how the tool should be used.

Reply:
The "main" field specifies the primary entry point of the program. Other programs could use this file when referencing some packages, for example, it would be used during the build and package process.